source/parser.hera

```diff
@@ -7366,6 +7366,8 @@ TypeLiteral
     return $0
   "void" NonIdContinue ->
     return { type: "VoidType", $loc, token: $1 }
+  "unique symbol" NonIdContinue ->
```

Comment:
I think `"unique" _ "symbol"` would be better; it seems TypeScript allows comments and newlines in between the two words.